app/assets/scss/_search-page.scss

```diff
@@ -27,3 +27,10 @@
     border: 1px solid $border-colour;
   }
 }
+
+.search-result-excerpt {
+  em {
```

Comment:
Yeah, sorry I missed that. I've added some code to make elasticsearch produce HTML that will let us use it: https://github.com/alphagov/digitalmarketplace-search-api/pull/33.

I've added some new SCSS that will use them instead (once that pull request is merged).